src/models/helpers/admin.rs

```diff
@@ -0,0 +1,91 @@
+use std::{collections::HashSet, num::ParseIntError, str::FromStr};
+
+use lazy_static::lazy_static;
```

Comment:
we might want to use `once_cell` instead :)

Reply:
Done, thanks!